### DISCUSSION TITLE
why resampler has many threads?

### DISCUSSION THREAD

**Discussion by zhngs** [PyAV-Org/PyAV#1469] [1. Help]
## Overview

<img width="1196" alt="WeChatWorkScreenshot_aed81e51-0080-4b49-b194-0b374cbe2d26" src="https://github.com/user-attachments/assets/e1d47308-25a3-4562-a212-16909cfa378b">

## Versions

- OS: linux centos 7
- PyAV runtime: 12.2.0

**Comment by XIAO-FAN-5257** [comment]
What command is this diagram executed by? Also, can the stack of threads inside the process be seen?
@zhngs

**Comment by XIAO-FAN-5257** [comment]
I also found this problem. Without adding a resampler, there are more than 60 threads for one stream. With a resampler added, there will be more than 120 threads.
